samples/crypto/sha256/sample.yaml

```
@@ -5,39 +5,39 @@ tests:
   sample.sha256:
     tags: introduction psa cc3xx
     platform_allow: >
-      nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160_ns
-      nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf9161dk_nrf9161 nrf9161dk_nrf9161_ns
-      nrf54h20dk_nrf54h20_cpuapp nrf54l15pdk_nrf54l15_cpuapp nrf54l15pdk_nrf54l15_cpuapp_ns
+      nrf5340dk/nrf5340/cpuapp/ns nrf5340dk/nrf5340/cpuapp nrf9160dk/nrf9160/ns
+      nrf9160dk/nrf9160 nrf52840dk/nrf52840 nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns
+      nrf54h20dk/nrf54h20/cpuapp nrf54l15pdk/nrf54l15/cpuapp nrf54l15pdk_nrf54l15_cpuapp_ns
     harness: console
     harness_config:
       type: multi_line
       regex:
         - ".*Example finished successfully!.*"
     integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp_ns
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf9160dk_nrf9160_ns
-      - nrf9160dk_nrf9160
-      - nrf9161dk_nrf9161_ns
-      - nrf9161dk_nrf9161
-      - nrf52840dk_nrf52840
-      - nrf54h20dk_nrf54h20_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
+      - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf9160dk/nrf9160/ns
+      - nrf9160dk/nrf9160
+      - nrf9161dk/nrf9161/ns
+      - nrf9161dk/nrf9161
+      - nrf52840dk/nrf52840
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54l15pdk_nrf54l15_cpuapp_ns
   # Build integration regression protection.
   sample.nrf_security.sha256.integration:
     build_only: true
     extra_args: CONFIG_BOOTLOADER_MCUBOOT=y
-    platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf52833dk_nrf52833
+    platform_allow: nrf5340dk/nrf5340/cpuapp/ns nrf52840dk/nrf52840 nrf52833dk/nrf52833
     integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp_ns
-      - nrf52840dk_nrf52840
-      - nrf52833dk_nrf52833
+      - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf52840dk/nrf52840
+      - nrf52833dk/nrf52833
   sample.newlib_libc.sha256:
     build_only: true
     extra_args: CONFIG_NEWLIB_LIBC=y
-    platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf54h20dk_nrf54h20_cpuapp
-    integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp_ns
-      - nrf52840dk_nrf52840
-      - nrf54h20dk_nrf54h20_cpuapp
+    platform_allow: nrf5340dk/nrf5340/cpuapp/ns nrf52840dk/nrf52840 nrf54h20dk/nrf54h20/cpuapp
+    integration/platforms:
```

```suggestion
    integration_platforms:
```

Please correct this due to Twister issue

---

done